app/assets/javascripts/cited_documents.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable camelcase */
+/* global CitedDocuments */
+/* global Blacklight */
+
+(function (global) {
+  var CitedDocuments;
+
+  CitedDocuments = {
+    init: function (el) {
+      var $el = $(el);
+      var data = $el.data();
+      data.documentids.forEach(function(documentId) {
+        $.getJSON(data.path, {
+          'f[id][]': documentId,
+          format: 'json',
+          rows: '1000'
```

Comment:
Since you already have the specific document ID, you can change the `rows` to 1.